### ISSUE TITLE
Proc List initialization fails

### ISSUE THREAD

**Issue by samulip** [hobarrera/caffeine-ng#8]
Hi, I think there might be an issue how list unpacking is used in main.py. Caffeine refused to start on my installation with two processes in the watchlist.

```
Traceback (most recent call last):
  File "/usr/local/bin/caffeine", line 22, in <module>
    main()
  File "/usr/local/lib/python3.4/dist-packages/caffeine/main.py", line 419, in main
    main = GUI(arguments["--preferences"])
  File "/usr/local/lib/python3.4/dist-packages/caffeine/main.py", line 222, in __init__
    name in self.__process_manager.get_process_list()])
TypeError: append() takes from 1 to 2 positional arguments but 3 were given
```

I fixed this with the following change. Hope this helps.

```diff
diff --git a/caffeine/main.py b/caffeine/main.py
index 426033c..d194e67 100644
--- a/caffeine/main.py
+++ b/caffeine/main.py
@@ -217,9 +217,8 @@ class GUI:
         self.selection.set_mode(Gtk.SelectionMode.MULTIPLE)

         self.proc_liststore = get("proc_liststore")
-        self.proc_liststore \
-            .append(*[[get_icon_for_process(name), name] for
-                      name in self.__process_manager.get_process_list()])
+        for name in self.__process_manager.get_process_list():
+                self.proc_liststore.append([get_icon_for_process(name), name])

         # time_menuitem = get("time_menuitem")
```

**Comment by hobarrera** [comment]
Thanks for spotting this out!

I think using `extend` might be more efficient and a simpler change:

```diff
diff --git a/caffeine/main.py b/caffeine/main.py
index 426033c..b33d9b9 100644
--- a/caffeine/main.py
+++ b/caffeine/main.py
@@ -218,7 +218,7 @@ class GUI:
 
         self.proc_liststore = get("proc_liststore")
         self.proc_liststore \
-            .append(*[[get_icon_for_process(name), name] for
+            .extend(*[[get_icon_for_process(name), name] for
                       name in self.__process_manager.get_process_list()])
 
         # time_menuitem = get("time_menuitem")
```

Can you confirm it fixes the issue as well?

**Comment by hobarrera** [comment]
Oh, it's not an actual python list, `extend` won't work.
Fixed via 8f4dbd6.
